README.md

```diff
@@ -26,16 +26,16 @@ See the wiki for detailed documentation.
 Ensure that docker is up to date and initialized.
 Spin up mysql locally:
 ```
-docker run -p 3306:3306 --name mysql -e MYSQL_ROOT_PASSWORD=rawls-test -e MYSQL_USER=rawls-test -e MYSQL_PASSWORD=rawls-test -e MYSQL_DATABASE=testdb -d mysql/mysql-server:5.7
+./docker/run-mysql.sh
```

Comment:
did you have any fiab mysql containers running at the same time?  i've found that when i have those up, even starting the mysql container the old way doesn't work and the container exits immediately.